travis/travis_transifex.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+from __future__ import print_function
+from __future__ import unicode_literals
+import os
+import sys
+from odoo_connection import context_mapping
+from test_server import setup_server, get_addons_path, \
+    get_server_path, get_addons_to_check
+from travis_helpers import yellow, yellow_light
+from txclib import utils, commands
+
+
+def main(argv=None):
+    """
+    Export translation files and push them to Transifex
+
+    The transifex password should be encrypted in .travis.yml
+    If not, export exits early.
+    """
+    if argv is None:
+        argv = sys.argv
+
+    transifex_user = os.environ.get("TRANSIFEX_USER")
+    transifex_password = os.environ.get("TRANSIFEX_PASSWORD")
+
+    if not transifex_user:
+        print(yellow_light("WARNING! Transifex user not defined- "
+              "exiting early."))
+        return 1
+
+    if not transifex_password:
+        print(yellow_light("WARNING! Transifex password not recognized- "
+              "exiting early."))
+        return 1
+
+    travis_home = os.environ.get("HOME", "~/")
```

Comment:
os.path.expanduser is never called on the string, using ~/ will likely result in a OSError, no such file or directory.